app/src/main/res/values/styles.xml

```diff
@@ -1,43 +1,82 @@
+	<style name="Widget.Kotatsu.Toolbar" parent="Widget.MaterialComponents.Toolbar">
+		<item name="android:paddingEnd">8dp</item>
+		<item name="popupTheme">@style/ThemeOverlay.Kotatsu</item>
+		<item name="titleTextAppearance">@style/TextAppearance.Kotatsu.ToolbarTitle</item>
 	</style>
 
-	<style name="AppActionMode" parent="Widget.AppCompat.ActionMode">
-		<item name="titleTextStyle">?attr/textAppearanceHeadline6</item>
-		<item name="subtitleTextStyle">?attr/textAppearanceSubtitle1</item>
-		<item name="background">@color/grey</item>
-	</style>
-
-	<style name="Widget.MaterialComponents.TabLayout.New">
+	<style name="Widget.Kotatsu.Tabs" parent="@style/Widget.MaterialComponents.TabLayout">
 		<item name="tabIndicatorFullWidth">false</item>
 		<item name="android:background">@drawable/tabs_background</item>
 		<item name="tabIndicator">@drawable/tab_indicator</item>
 		<item name="tabIconTint">@color/tab_text</item>
 		<item name="tabSelectedTextColor">?colorPrimary</item>
 		<item name="tabTextColor">@color/tab_text</item>
-		<item name="tabTextAppearance">@style/TextAppearance.Design.Tab.New</item>
+		<item name="tabTextAppearance">@style/TextAppearance.Kotatsu.Tab</item>
+		<item name="tabIndicatorAnimationMode">elastic</item>
+	</style>
+
+	<style name="Widget.Kotatsu.NavigationView" parent="@style/Widget.MaterialComponents.NavigationView">
+		<item name="itemBackground">@drawable/navigation_item_background</item>
+		<item name="itemHorizontalPadding">@dimen/nav_item_horizontal_padding</item>
 	</style>
 
-	<style name="TextAppearance.Design.Tab.New">
+	<style name="Widget.Kotatsu.Chip" parent="Widget.MaterialComponents.Chip.Action">
+		<item name="chipStrokeWidth">1dp</item>
+		<item name="chipStrokeColor">?attr/colorAccent</item>
+		<item name="chipBackgroundColor">@android:color/transparent</item>
+		<item name="chipIconTint">?attr/colorPrimary</item>
+		<item name="android:textColor">?attr/colorAccent</item>
+
+		<!-- Smaller text/height -->
+		<item name="android:textSize">12sp</item>
+
+		<!-- Custom chip states -->
+		<item name="android:clickable">true</item>
+		<item name="android:focusable">true</item>
+	</style>
+
+	<style name="Widget.Kotatsu.ToggleButton" parent="Widget.MaterialComponents.Button.OutlinedButton">
+		<item name="android:checkable">true</item>
+		<item name="android:gravity">center_vertical|start</item>
+		<item name="iconPadding">16dp</item>
+		<item name="android:paddingTop">10dp</item>
+		<item name="android:paddingBottom">10dp</item>
+	</style>
+
+	<style name="Widget.Kotatsu.ActionMode" parent="Widget.AppCompat.ActionMode">
+		<item name="titleTextStyle">?attr/textAppearanceHeadline6</item>
+		<item name="subtitleTextStyle">?attr/textAppearanceSubtitle1</item>
+	</style>
+
+	<!-- TextAppearance -->
+
+	<style name="TextAppearance.Kotatsu.ToolbarTitle" parent="@style/TextAppearance.Widget.AppCompat.Toolbar.Title">
+		<item name="android:textSize" tools:ignore="SpUsage">20dp</item>
```

Comment:
Why `dp` instead of `sp`?

Reply:
Whoops, sorry, I didn't notice this, in the Google I/O repository, for some reason, `dp` is presented, not `sp`, which is really weird